packages/realm/src/app-services/SyncConfiguration.ts

```diff
@@ -115,6 +115,7 @@ export type BaseSyncConfiguration = {
   /** @internal */
   _sessionStopPolicy?: SessionStopPolicy;
   clientReset?: ClientResetConfig;
+  cancelWaitsOnNonFatalError?: boolean;
```

Comment:
Question is if we want to use `Nonfatal` (instead of `NonFatal`) like Core instead?

Reply:
This was not exposed before, so we can do as we want with naming. I thought that `Nonfatal` seems an error compared to `NonFatal` if we want to keep camel case. I'm not strongly opposed though 😁